### PR TITLE
Granularize cured hides and pelts

### PR DESCRIPTION
**Pull request by Podesta**
#### Summary

```SUMMARY: Balance "Granularize cured hides and pelts"```

#### Purpose of change
After the introduction of granular pieces of raw hides and pelts in #27399 a common problem is that you often don't get the required amount (50) of raw bits to craft a cured hide. Since the raw bits spoil, you often end up wasting massive amounts of hides. More often than not, you end up with nothing. This tries to mitigate the issue.

#### Describe the solution

- Granularize the curing part of process. Instead of 50 raw bits for 1 cured item, now you need 1 raw bit to create 1 cured bit, and 50 cured bits to 1 tanning bit. It only moves the granular aspect one step further, so the hides and pelts don't spoil.

- Since there is no way to scale the amount of salt based on the current recipe, I didn't change the salt recipe (it still takes the 50 raw bits), and instead mainlined the recipe from the `More Survival Tools`, that requires an open fire instead, and can be easily scaled. The problem is that now, the recipes that uses salt uses the `"result_mult": 50`, which is very spammy. A better solution would be great.

- Remove the raw human hide (`raw_hleather`) from the couple of recipes that had it, since humans seems to be dropping regular raw hides anyways.

#### Describe alternatives you've considered
Make so that raw pelts and hides no longer spoil, so the user can have enought time to gather the necessary amount.

#### Additional context
The objective of this PR is to mitigate the [problems and confusion](https://discourse.cataclysmdda.org/t/bug-with-cured-pelt-recipe-in-build-9374/20890/6) that the new granular system introduced. I made no changes to the amount of hide butchering gives, how reasonable the crafting process is, how long it should take, if other recipes that used the cured hide are reasonable, etc, since my knowledge of the area is very lacking. I kept the same proportions that were already in place, and simply re scaled them.

### **IMPORTANT:** Since all cured hides and pelts now weight and have a volume that is 1/50 of what they used to be, and since now you require 50 of them to craft a single tanning piece, if you have multiple cured hides waiting to be tanned, you will essentially lose most of them. You can fix this in two ways:

1. Use the debug menu and spawn yourself 49 cured hides or pelts for each cured hide you currently have.

2. Not update until you started tanning all of the cured hides/pelts you have.

### CODE REVIEW

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/bug-with-cured-pelt-recipe-in-build-9374/20890/7

**Comment by Podesta** [comment]
Jenkins rebuild

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -3704,11 +3702,26 @@
...
     "difficulty": 1,
-    "time": "20 m",
+    "time": "20 s",
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "cured_pelt", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "cured_hide",
```
Comment (kevingranade) [inline data/json/recipes/recipe_others.json]: I can't find a resource that indicates that drying via heating is a workable method for preserving hide.

**Comment by Podesta** [comment]
Fair enough. I believe the mod was abstracting the process of letting it dry in the air by using fire. Here are the changes I'll make:

- Separate the curing from the cleaning process. You now must clean the skin before curing it, and curing is no longer mandatory for further processing. It has no water requirements because there is no way to sanely scale it. The water requirements is pushed to the tanning process.

- Add new recipe for air curing a leather hide. Has no real requirements, other than waiting the time it takes to process (a few days). I removed the recipe that used fire as abstraction to this process.

- Salt curing recipe now scaled for 5 units of raw hides instead of 50.

- You can now tan the hide with either cured or clean hides. The process now takes 6 days instead of the current 8 hours to complete. Salt is not needed for tanning. The fat requirements is now part of the "OR" with the other tannins. The tools required now are containing and hammering. The recipe that uses lye remains unchanged (other than adding the clean hides).
  - Not sure if it is preferable to have the transformation time in turns or the new hours system. What scales better with non default season length?

@kevingranade  I'm not sure about the formatting. Do you want me to change it back to the components on the same line, like your commit? I believe the [style guide](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md) is outdated, and the formatting tool didn't help much. And the file itself is a mixture of both the more readable multiple lines, and the single line format.

**Comment by kevingranade** [comment]
Please put that in a new PR, the current one is about granularity, let's just get that done and worry about all this new stuff later.
The formatting isn't just a general style it is precisely what the tool outputs.

**Comment by Podesta** [comment]
The problem is that you can't go to a one to one granular recipe without a new recipe. Since the curing process is currently mandatory for tanning a leather in the game, you would need 5 grams of salt to cure 15 grams of raw leather, which becomes 12 grams of cured leather.

The real problem I'm trying to fix is the fact that currently you waste pretty much all of the leather you get in the game. Even if you kill and butcher a full deer you are unable to do anything with the leather you get from it. Granularization is part of the fix, but not the whole fix. If you prefer I can change the summary and the title.

As for the formatting tool, if I put this into the tool:

```
  {
    "type": "recipe",
    "result": "tanning_hide",
    "category": "CC_OTHER",
    "subcategory": "CSC_OTHER_MATERIALS",
    "skill_used": "survival",
    "difficulty": 3,
    "time": "48 m",
    "batch_time_factors": [ 20, 3 ],
    "autolearn": true,
    "qualities": [
      { "id": "HAMMER", "level": 1 },
      { "id": "CONTAIN", "level": 1 }
    ],
    "components": [
      [ [ "water_clean", 2 ], [ "water", 2 ] ],
      [ [ "tanbark", 1 ], [ "acorns", 2 ], [ "hops", 2 ], [ "pine_bough", 6 ], [ "tallow", 1 ], [ "tallow_tainted", 1 ], [ "lard", 1 ] ],
      [ [ "clean_hide", 50 ], [ "cured_hide", 50 ] ]
    ]
  }
```

The output I get is this:
```

{
  "type": "recipe",
  "result": "tanning_hide",
  "category": "CC_OTHER",
  "subcategory": "CSC_OTHER_MATERIALS",
  "skill_used": "survival",
  "difficulty": 3,
  "time": "48 m",
  "batch_time_factors": [
    20,
    3
  ],
  "autolearn": true,
  "qualities": [
    { "id": "HAMMER", "level": 1 },
    { "id": "CONTAIN", "level": 1 }
  ],
  "components": [
    [ [ "water_clean", 2 ], [ "water", 2 ] ],
    [
      [ "tanbark", 1 ],
      [ "acorns", 2 ],
      [ "hops", 2 ],
      [ "pine_bough", 6 ],
      [ "tallow", 1 ],
      [ "tallow_tainted", 1 ],
      [ "lard", 1 ]
    ],
    [ [ "clean_hide", 50 ], [ "cured_hide", 50 ] ]
  ]
}
```
And trying to add more than a recipe at a time I get a syntax error.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/fix-pelts-already/21400/2

**Comment by anothersimulacrum** [comment]
You need to lint the entire file at the same time - you don't lint each recipe individually, you put all the recipes in the file, then copy and paste the entire file into the tool and lint it.

File: data/json/items/generic.json [hunk 2]
```diff
@@ -2226,19 +2226,28 @@
...
     "name_plural": "cured hides",
-    "description": "A rolled up animal hide which has been scraped of extraneous hair and flesh and treated to prevent decay.  It still requires tanning to become usable leather.",
-    "price": 2000,
+    "description": "A rolled up animal hide which has been preserved by either salting or drying.  It still requires tanning to become usable leather.",
+    "price": 40,
     "material": [ "flesh" ],
     "flags": [ "NO_SALVAGE" ],
-    "weight": 600,
-    "volume": 3,
+    "weight": 12,
```
Comment (anothersimulacrum) [inline data/json/items/generic.json]:
```suggestion
    "weight": "12 g",
```

File: data/json/items/generic.json [hunk 3]
```diff
@@ -2257,19 +2266,28 @@
...
     "name_plural": "cured pelts",
-    "description": "A rolled up animal hide which has been scraped of extraneous hair and flesh and treated to prevent decay.  It still requires tanning to become usable fur.",
-    "price": 2000,
+    "description": "A rolled up animal hide which has been preserved by either salting or drying.  It still requires tanning to become usable fur.",
+    "price": 40,
     "material": [ "fur", "flesh" ],
     "flags": [ "NO_SALVAGE" ],
-    "weight": 684,
-    "volume": 4,
+    "weight": 14,
```
Comment (anothersimulacrum) [inline data/json/items/generic.json]:
```suggestion
    "weight": "14 g",
```

File: data/json/items/generic.json [hunk 4]
```diff
@@ -2283,7 +2301,7 @@
     "description": "A folded sheet of leather made from carefully tanned animal hide, with the fur still intact.  Can be cut up or used as is.",
     "price": 5000,
     "material": [ "fur" ],
-    "weight": 684,
+    "weight": 700,
```
Comment (anothersimulacrum) [inline data/json/items/generic.json]:
```suggestion
    "weight": "700 g",
```

File: data/json/items/tools.json [hunk 5]
```diff
@@ -5662,6 +5662,31 @@
...
     "flags": [ "ALLOWS_REMOTE_USE" ]
   },
+  {
+    "id": "air_curing_hide",
+    "type": "TOOL",
+    "category": "spare_parts",
+    "name": "air drying leather hide",
+    "name_plural": "air drying leather hides",
+    "description": "A clean animal hide which is being left to dry to prevent decay. Keep it away from the rain and water. You will be able to activate it when it has finished drying.",
+    "weight": 12,
```
Comment (anothersimulacrum) [inline data/json/items/tools.json]:
```suggestion
    "weight": "12 g",
```

File: data/json/items/tools.json [hunk 6]
```diff
@@ -5680,9 +5705,34 @@
...
+    "flags": [ "NO_SALVAGE" ]
+  },
+  {
+    "id": "air_curing_pelt",
+    "type": "TOOL",
+    "category": "spare_parts",
+    "name": "air drying pelt",
+    "name_plural": "air drying pelts",
+    "description": "A clean animal fur which is being left to dry to prevent decay. Keep it away from the rain and water. You will be able to activate it when it has finished drying.",
+    "weight": 12,
```
Comment (anothersimulacrum) [inline data/json/items/tools.json]:
```suggestion
    "weight": "12 g",
```

**Comment by ZhilkinSerg** [comment]
Probably obsoleted by #34291.

**Pull request closed without merging**